layouts/shortcodes/rum/session_replay/mobile/privacy_options.mdoc.md

```diff
@@ -1199,6 +1441,18 @@ This includes:
 {% /if %}
 <!-- end React Native -->
 
+<!-- Flutter -->
+{% if equals($platform, "flutter") %}
+**Shapes**
+- Background of selected option in Segment
+- Circle surrounding the selected date in a Date Picker
+- Thumb of a Slider
```

Comment:
Is "Thumb" a commonly used word in this context? Perhaps "selector" would be more generic?